[2024-04-15T10:06:26.531+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-15T10:06:26.587+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: prioritas2.write_to_txt scheduled__2024-04-14T00:00:00+00:00 [queued]>
[2024-04-15T10:06:26.666+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: prioritas2.write_to_txt scheduled__2024-04-14T00:00:00+00:00 [queued]>
[2024-04-15T10:06:26.675+0000] {taskinstance.py:2303} INFO - Starting attempt 4 of 6
[2024-04-15T10:06:26.808+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): write_to_txt> on 2024-04-14 00:00:00+00:00
[2024-04-15T10:06:26.847+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=395) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-15T10:06:26.855+0000] {standard_task_runner.py:63} INFO - Started process 408 to run task
[2024-04-15T10:06:26.855+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'prioritas2', 'write_to_txt', 'scheduled__2024-04-14T00:00:00+00:00', '--job-id', '337', '--raw', '--subdir', 'DAGS_FOLDER/prioritas2_dag.py', '--cfg-path', '/tmp/tmp4gl_5zld']
[2024-04-15T10:06:26.868+0000] {standard_task_runner.py:91} INFO - Job 337: Subtask write_to_txt
[2024-04-15T10:06:27.095+0000] {task_command.py:426} INFO - Running <TaskInstance: prioritas2.write_to_txt scheduled__2024-04-14T00:00:00+00:00 [running]> on host 2354a2c4893f
[2024-04-15T10:06:27.841+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abdan' AIRFLOW_CTX_DAG_ID='prioritas2' AIRFLOW_CTX_TASK_ID='write_to_txt' AIRFLOW_CTX_EXECUTION_DATE='2024-04-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-14T00:00:00+00:00'
[2024-04-15T10:06:27.864+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-15T10:06:27.865+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-15T10:06:27.866+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: write_to_txt() missing 1 required positional argument: 'data'
[2024-04-15T10:06:27.953+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=prioritas2, task_id=write_to_txt, execution_date=20240414T000000, start_date=20240415T100626, end_date=20240415T100627
[2024-04-15T10:06:28.116+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 337 for task write_to_txt (write_to_txt() missing 1 required positional argument: 'data'; 408)
[2024-04-15T10:06:28.239+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-04-15T10:06:28.380+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-15T10:06:28.412+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
